style.css

```diff
@@ -207,4 +207,232 @@ body {
 .container a:hover {
   background-color: bisque;
   color: black;
+}
+
+ul {
+  display: grid;
+  grid-template-columns: repeat(3, 40px);
+  gap: 0;
+  margin: 5px 0;
+  list-style: none;
+}
+
+.skills {
+  text-decoration: none;
+  background-color: #ebebff;
+  text-align: center;
+  padding: 5px 7px;
+  border-radius: 10px;
+  font-size: 13px;
+}
+
+/* stylelint-disable-next-line no-duplicate-selectors */
+.container-section {
+  background: #e5e5e5;
+  width: 350px;
+  border-radius: 20px;
+}
+
+.grid-container-card {
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  gap: 70px;
+  margin-bottom: 10px;
+}
+
+.grid-card {
+  background-color: white;
+  padding: 15px 5px;
+  margin: 0 20px;
+  border-radius: 10px;
+}
+
+.responsive-backgroundImage {
+  height: 200px;
+  width: 300px;
+  margin: 0 auto;
+}
+
+.responsive-backgroundImage img {
+  height: 100%;
+  width: 100%;
+}
+
+.identification {
+  margin: 15px 10px;
+  font-family: sans-serif;
+}
+
+.identification h3 {
+  font-weight: 600;
+  font-size: 30px;
+  margin: 10px 0;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  font-weight: 800;
+}
+
+.flex-container .title-bolder {
+  font-size: 11px;
+  margin: 10px 0;
+}
+
+.flex-container .title-non-bolder {
+  font-size: 11px;
+  margin: 10px 0;
+  color: rgb(148, 147, 147);
+}
+
+.flex-container .point {
+  width: 5px;
+  height: 5px;
+  background-color: gray;
+  border-radius: 50%;
+  margin: 0 15px;
+  color: rgb(148, 147, 147);
+}
+
+.text p {
+  margin: 5px 0;
+  color: rgb(63, 64, 62);
+}
+
+/* Adding padding to the button. */
+.btn {
+  padding: 8px;
+  border-radius: 5px;
+  margin: 8px 0;
+  transition: all 0.5s;
+  border: 2px solid #396df2;
+  color: #396df2;
+  font-size: 18px;
+  font-weight: 500;
+}
+
+.btn:hover {
+  background: rgb(14, 12, 12);
+  color: beige;
+}
+
+.about {
+  background-color: #fff;
+  margin-top: 40px;
+  padding: 20px;
+  border-top-right-radius: 15%;
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
good job spotting and using the border-top-right-radius in this section but you need to make it match with the design.

The border color below the framework and skills does not match the design, kindly fix it.

Also, Kindly fix the Images link. I dropped the instruction in the other review

[optional] It will be great if you removed the extra color in the project background and increase the width of the main content to cover the screen based on the design, remember to use percentages for the width 😄 

![image](https://user-images.githubusercontent.com/43564324/159125894-12b596e9-e4c6-49c8-b24b-814217da8ee7.png)